Reports: Add section on freezing a run state (#1062)
wandb/docs: content/guides/core/reports/edit-a-report.md

{{% /tab %}}
{{< /tabpane >}}

> ## Preserve run set display

Reviewer: Why not "Freeze a run set"?

> Reports automatically update run sets to show the latest data from the project. You can preserve the run set in a report by *freezing* that run set. When you freeze a run set, you preserve the state of the run set at a point in time.

Reviewer: ```suggestion
A report automatically updates run sets to show the latest data from the project. Freeze a run set to prevent these updates and preserve the state of the run set in a report at a point in time.
```

To freeze a run set:

1. Navigate to your report.

> 2. Click the snowflake icon located in the run set's panel grid, next to the **Filter** button.

Reviewer: ```suggestion
To freeze a run set when viewing a report, click the snowflake icon in its panel grid near the **Filter** button.
```

{{< img src="/images/reports/freeze_runset.png" alt="" >}}

> 

Reviewer: ```suggestion
```
I wonder how this happened?!
